tests/istio_testcases/expected_output/istio_ingress_test_connectivity_map.txt

```diff
@@ -1,8 +1,8 @@
 final fw rules for query: connectivity, config: istio-ingress:
 src: 0.0.0.0/0 dst_ns: [default,prod,qa] dst_pods: [*] conn: All connections
 src_ns: [default] src_pods: [*] dst_ns: [default] dst_pods: [istio-ingressgateway] conn: All connections
-src_ns: [default] src_pods: [*] dst_ns: [prod] dst_pods: [ratings-v1-857bb87c57] conn: TCP {'dst_ports': '9080', 'paths': '/reviews/[\\--9A-Z_a-z]*', 'hosts': 'eu.bookinfo.com, uk.bookinfo.com'}
-src_ns: [default] src_pods: [*] dst_ns: [qa] dst_pods: [*] conn: TCP {'dst_ports': '7777', 'hosts': 'eu.bookinfo.com, uk.bookinfo.com'}
+src_ns: [default] src_pods: [*] dst_ns: [prod] dst_pods: [ratings-v1-857bb87c57] conn: TCP {'dst_ports': '9080', 'hosts': 'eu.bookinfo.com, uk.bookinfo.com, reviews.prod.svc.cluster.local', 'paths': '/reviews/*'}
+src_ns: [default] src_pods: [*] dst_ns: [qa] dst_pods: [*] conn: TCP {'dst_ports': '7777', 'hosts': 'eu.bookinfo.com, uk.bookinfo.com, reviews.prod.svc.cluster.local'}
```

Comment:
Why do we have a new host here?

Reply:
Due to a fix that I've made in `GenericIngressLikeYamlParser` : the function `parse_regex_host_value` was not handling correctly the host regex when it was just `*` , and after this fix an additional host from the VirtualService was matched by the hosts regex from the Gateway.